Sources/SpeziAccount/AccountValue/Configuration/ConfiguredAccountKey.swift

```diff
@@ -38,23 +38,31 @@ public struct ConfiguredAccountKey {
     /// - Parameter keyPath: The `KeyPath` referencing the ``AccountKey``.
     /// - Returns: Returns the ``AccountKey`` configuration.
     public static func requires<Key: AccountKey>(_ keyPath: KeyPath<AccountKeys, Key.Type>) -> ConfiguredAccountKey {
-        .init(configuration: AccountKeyConfigurationImpl(keyPath, type: .required))
+        .init(configuration: AccountKeyConfigurationImpl(keyPath, requirement: .required))
     }
 
     /// Configure an ``AccountKey`` as ``AccountKeyRequirement/collected``.
     /// - Parameter keyPath: The `KeyPath` referencing the ``AccountKey``.
     /// - Returns: Returns the ``AccountKey`` configuration.
     @_disfavoredOverload
     public static func collects<Key: AccountKey>(_ keyPath: KeyPath<AccountKeys, Key.Type>) -> ConfiguredAccountKey {
-        .init(configuration: AccountKeyConfigurationImpl(keyPath, type: .collected))
+        .init(configuration: AccountKeyConfigurationImpl(keyPath, requirement: .collected))
     }
 
     /// Configure an ``AccountKey`` as ``AccountKeyRequirement/supported``.
     /// - Parameter keyPath: The `KeyPath` referencing the ``AccountKey``.
     /// - Returns: Returns the ``AccountKey`` configuration.
     @_disfavoredOverload
     public static func supports<Key: AccountKey>(_ keyPath: KeyPath<AccountKeys, Key.Type>) -> ConfiguredAccountKey {
-        .init(configuration: AccountKeyConfigurationImpl(keyPath, type: .supported))
+        .init(configuration: AccountKeyConfigurationImpl(keyPath, requirement: .supported))
+    }
+    
+    /// Configure an ``AccountKey`` as ``AccountKeyRequirement/hidden``.
+    /// - Parameter keyPath: The `KeyPath` referencing the ``AccountKey``.
+    /// - Returns: Returns the ``AccountKey`` configuration.
+    @_disfavoredOverload
```

Comment:
`@_disfavoredOverload` is not necessary. Only if you add an overload for the `RequiredAccountKey` as well (we use it to make sure that the deprecation warning is printed). However, we could think about adding an overload for `hidden` (or whatever we call it) for `RequiredAccountKey` as well, adding a deprecation warning that this probably doesn't make sense (after all it is state as required). Implementation would stay the same.

Reply:
I added the other extension as well now - calling both `manual`

Reply:
nice, thank you 👍